stack.yml

```diff
@@ -47,7 +47,7 @@ functions:
   git-tar:
     lang: dockerfile
     handler: ./git-tar
-    image: functions/of-git-tar:0.11.2
+    image: functions/of-git-tar:0.12.0
```

Comment:
Can we sync ofc-bootstrap?

Reply:
Actually changing this file with a new set of code to be merged is hard because of timing. I have two push a new image immediately after merging your PR and if someone pulls in that window it may break for them.

It's harder for you but probably we should do code and config changes separately.

Reply:
Sounds reasonable, there is no problem 👍

Reply:
Reverted that 👍

Reply:
Added in different PR